core/src/test/scala/org/apache/spark/ui/storage/StorageTabSuite.scala

```diff
@@ -22,6 +22,7 @@ import org.apache.spark.Success
 import org.apache.spark.executor.TaskMetrics
 import org.apache.spark.scheduler._
 import org.apache.spark.storage._
+import org.apache.spark.SparkConf
```

Comment:
nit: import ordering, should go above the rest of the spark imports (b/c class imports sort before package imports).  Also while you're touching this, there should be a blank line between the scalatest and spark imports